test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetRestoreTests.cs

```diff
@@ -1468,6 +1468,62 @@ public void GenerateRestoreGraphFile_StandardAndStaticGraphRestore_AreEquivalent
             }
         }
 
+        [Theory]
+        [InlineData(true, true)]
+        [InlineData(true, false)]
+        [InlineData(false, false)]
```

Comment:
Am I right that `false, true` is omitted because the spec factory only makes sense with static graph restore?

Reply:
yes, that's right. When I add PackageSpecFactory support to msbuild restore, then part of "adding unit tests" will be adding a new InlineData line to all the tests where they're currently missing.